### PR TITLE
Add all inputstreams as dependencies and fix debian control file

### PR DESCRIPTION
**Pull request by phunkyfish**
v4.13.2
- Fixed: Install all inputstream dependencies by default.
- Fixed: Debian control files for run-time dependencies.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-Homepage: <http://kodi.tv>
+Homepage: <https://kodi.tv>
 
 Package: kodi-pvr-iptvsimple
 Section: libs
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
+Recommends: kodi-inputstream-adaptive,
+            kodi-inputstream-ffmpegdirect,
+            kodi-inputstream-rtmp
```
Comment (phunkyfish) [inline debian/control:16]: @basilgello Is this good?

File: pvr.iptvsimple/addon.xml.in [hunk 2]
```diff
@@ -167,6 +169,10 @@
       <icon>icon.png</icon>
     </assets>
     <news>
+v4.13.2
+- Fixed: Install all inpustream dependencies by default.
+- Fixed: Debian control files for run-time depednencies.
```
Comment (wsnipex) [inline pvr.iptvsimple/addon.xml.in]:
```suggestion
- Fixed: Debian control files for run-time dependencies.
```

File: pvr.iptvsimple/addon.xml.in [hunk 3]
```diff
@@ -167,6 +169,10 @@
       <icon>icon.png</icon>
     </assets>
     <news>
+v4.13.2
+- Fixed: Install all inpustream dependencies by default.
```
Comment (wsnipex) [inline pvr.iptvsimple/addon.xml.in]:
```suggestion
- Fixed: Install all inputstream dependencies by default.
```

File: pvr.iptvsimple/changelog.txt [hunk 4]
```diff
@@ -1,3 +1,7 @@
+v4.13.2
+- Fixed: Install all inpustream dependencies by default.
+- Fixed: Debian control files for run-time depednencies.
```
Comment (wsnipex) [inline pvr.iptvsimple/changelog.txt]:
```suggestion
- Fixed: Debian control files for run-time dependencies.
```

File: pvr.iptvsimple/changelog.txt [hunk 5]
```diff
@@ -1,3 +1,7 @@
+v4.13.2
+- Fixed: Install all inpustream dependencies by default.
```
Comment (wsnipex) [inline pvr.iptvsimple/changelog.txt]:
```suggestion
- Fixed: Install all inputstream dependencies by default.
```

**Review by wsnipex — COMMENTED** [review]
a few typos, see suggestions

**Comment by basilgello** [comment]
Do we use rtmp within pvr anywhere? If yes, apart from typo fixes Wolfgang pointed faster than me, fine.

**Comment by phunkyfish** [comment]
Some users do yes. Someone requested it before but I just didn't know how to do it.

**Pull request merged**